### PR TITLE
Updated readme with Google Cloud config info. 

### PR DESCRIPTION
**Pull request by mikesaltsman**
Firstly, thank you for the amazing plugin for Shrine. It helps immensely! 

I also removed a dead link and added a link to creating a service account on the GCS repo. I'm hoping this makes it a little clearer for new users trying to tie into Google Cloud Storage. 

Thanks again!

### CODE REVIEW

**Comment by quartzmo** [comment]
@mikesaltsman Thank you for adding this[`Google::Cloud.configure`](http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-core/latest/Google/Cloud.html#configure-class_method) example for [google-cloud-storage](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-storage). (There is also a [`Google::Cloud::Storage.configure`](http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-storage/latest/Google/Cloud/Storage.html#configure-class_method) method that adds a few things.)

Would it also make sense to link to the [google-cloud-storage authentication guide](http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-storage/latest/file.AUTHENTICATION.html) in this README? The authentication guide has general instructions for creating a service account and generating a JSON keyfile.

@renchap Thanks for creating this integration with Shrine and GCS! If there's anything you need changed or added in google-cloud-storage, please don't hesitate to open a PR or an issue.

File: README.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ GOOGLE_CLOUD_PROJECT=my-project
...
 
+Be sure you are setting the Google Cloud config using the variables just created.
+
+```sh
+Google::Cloud.configure do |config|
+    config.project_id = GOOGLE_CLOUD_PROJECT
+    config.keyfile = GOOGLE_CLOUD_KEYFILE
+end
+```
+
```
Comment (renchap) [inline README.md]: I am not sure I understand this part. This section is about running the test suite of this Shrine plugin, not about how to use it.

**Comment by renchap** [comment]
@quartzmo We can definitely add a link to the authentication guide.

But after reviewing this PR the second part changes the instructions intended for running this gem's test suite, not normal usage, and I am not sure this is the initial intention.

Do you have any stable URL that we can link to in the README's authentication section? Should it be 
http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-storage/latest/file.AUTHENTICATION.html as you proposed?

**Comment by quartzmo** [comment]
> We can definitely add a link to the authentication guide.

Thank you! This is the stable URL: http://googleapis.github.io/google-cloud-ruby/docs/google-cloud-storage/latest/file.AUTHENTICATION.html

> he second part changes the instructions intended for running this gem's test suite, not normal usage, and I am not sure this is the initial intention.

I agree. The intention of @mikesaltsman's PR was to add an authentication example for end users, not for developers or users running the test suite.

**Comment by renchap** [comment]
README updated!

**Pull request closed without merging**